ci/nightly/pipeline.yml

```diff
@@ -77,7 +77,7 @@ steps:
           - { value: parallel-workload-dml }
           - { value: parallel-workload }
           - { value: parallel-workload-100-threads }
-          #- { value: parallel-workload-cancel }
```

Comment:
I guess this PR contains the changes of the previous PR as well and needs a rebase after the merge of that?

Reply:
I'm not sure if Github is smart enough to adapt the diff after the other one is merged, if not I can rebase.

Reply:
No worries, the PR is small enough that I understand what the new changes are.